src/components/HeroIcon/HeroIcon.tsx

```diff
@@ -10,18 +10,22 @@ export interface HeroIconProps {
   outline?: boolean;
   color?: 'black' | 'white' | 'red' | 'grey' | 'dark-grey';
   size?: '4' | '6' | '8' | '12' | '16' | '20';
+  className?: string;
 }
 
 export const HeroIcon = ({
   icon,
   outline = false,
-  color = 'black',
+  color,
   size = '8',
+  className,
```

Comment:
Let's drop className from here and line 28. We can find a better way in the component file to handle the spacing

Reply:
removed classname form hero